actions/installer/src/index.ts

```diff
@@ -0,0 +1,141 @@
+import * as core from "@actions/core";
+import * as exec from "@actions/exec";
+import * as io from "@actions/io";
+import * as tc from "@actions/tool-cache";
+import * as crypto from "crypto";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+const LATEST_VERSION = "v1.3.0";
+
+const VERIFIER_V130_SHA256 =
```

Comment:
nit: make the name indicate it's used for bootstrapping. Set another global here to specify the bootstrap version or URL (see other comment).
```suggestion
const BOOTSTRAP_VERIFIER_SHA256 =
```